### PR TITLE
adapt document to priority not changing through the DODAG

### PR DESCRIPTION
**Pull request by mcr**

### CODE REVIEW

File: 6tisch-roll-enrollment-priority.mkd [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
 title: Controlling Secure Network Enrollment in RPL networks
 abbrev: join-metric
-docname: draft-ietf-roll-enrollment-priority-07
+docname: draft-ietf-roll-enrollment-priority-06
```
Comment (nyrahul) [inline 6tisch-roll-enrollment-priority.mkd:4]: Why is the new draft version lower than previous?

File: 6tisch-roll-enrollment-priority.mkd [hunk 2]
```diff
@@ -79,30 +79,32 @@ friendly router as a Join Proxy.
...
 Depending on the network policy, the size of the DODAG may or may not affect the minimum enrollment priority.
 It would be limiting its value to enforce that one is proportional to the other.
-This is why the current size of the DODAG is advertised separately in the new option.
+The current size of the DODAG is therefore also advertised in the new option.
 
 As explained in {{!RFC9032}}, higher values decrease the likelyhood of an unenrolled node sending enrollment traffic via this path.
 
 A network operator can set this value to the maximum value allowed, effectively disable all new enrollment traffic.
 
+Updates to this option propogate slowly through the network according to the trickle algorithm that periodically sends updates.
```
Comment (nyrahul) [inline 6tisch-roll-enrollment-priority.mkd:105]: If we assume that we never reset the DIO trickle timer, the option propagation in worst case may take hours or even days to propagate depending on DIO trickle timer parameters.

File: 6tisch-roll-enrollment-priority.mkd [hunk 3]
```diff
@@ -113,15 +115,15 @@ In the context of the {{!RFC6550}} RPL protocol, the term (2)"Join" has an alter
...
 
 
 # Protocol Definition
 
 With this specification, the following option is defined for transmission in the DIO issued by the DODAG root and it MUST be propagated down the DODAG.
 
-A 6LR which would otherwise be willing to act as a *Join Proxy*, will examine the minimum priority field, and to that number, add any additional local consideration (such as upstream congestion).
+A 6LR which would otherwise be willing to act as a *Join Proxy*, will examine the minimum priority field, and to that number, add any additional local consideration (such as upstream congestion, number of NCE slots available, etc.)
 
 The Enrollment Priority can only be increased by each 6LR in value, to the maximum value of 0x7f.
```
Comment (nyrahul) [inline 6tisch-roll-enrollment-priority.mkd:128]: Are we allowing the 6LRs to change the Enrollment priority? Based on @mcr mail, it was not allowed.

**Review by nyrahul — COMMENTED** [review]
Please find my review inline. Thanks

File: 6tisch-roll-enrollment-priority.mkd (hunk 1 above)
Comment (mcr) [reply 6tisch-roll-enrollment-priority.mkd:4]: I think that I goofed incrementing the number too many times.  -05 is what is posted.

File: 6tisch-roll-enrollment-priority.mkd (hunk 3 above)
Comment (mcr) [reply 6tisch-roll-enrollment-priority.mkd:128]: We aren't changing the global enrollment priority, but in translating this RPL metric to the one in RFC9032, the local node is allowed to take into consideration local conditions. Perhaps this isn't worded well.

**Comment by pthubert** [comment]
We need to provide the math or at least indicate the the OF’ that computes the priority in the beacon from the min I. The DIO MUST be homogeneous across DODAGs of the RPL instance…


A bientôt;

Pascal

**Comment by mcr** [comment]
> We need to provide the math or at least indicate the the OF’ that computes the priority in the beacon from the min I. The DIO MUST be homogeneous across DODAGs of the RPL instance… A bientôt; Pascal

Are you saying that you want two nodes, from two different vendors, to emit the same Enhanced Beacon priority, given the same input from the min I?
(I'm not saying this is bad, I'm wondering if we know enough to write this down at this point)

**Comment by pthubert** [comment]
Yes. Because a node that may join dag à or b should be able to judge on equal terms


A bientôt;

Pascal

**Comment by mcr** [comment]
already merged.

**Pull request closed without merging**